src/phone-input/__tests__/phone-input.e2e.js

```diff
@@ -63,6 +70,23 @@ describe('PhoneInput', () => {
     });
   });
 
+  it('allows a user select a country using the keyboard', async () => {
+    // click select
+    await page.click(selectors.phoneInputSelect);
+    // verify dropdown is open
+    await page.waitFor(selectors.phoneInputSelectDropdown);
+
+    await page.keyboard.type('United Kingdom');
```

Comment:
can you please add a test showing that if I type in `United` and press arrow down, I reach `United Arab Emirates`?